pom.xml

```diff
@@ -4,12 +4,12 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.microsoft.azure</groupId>
-    <artifactId>azure-spring-boot-starters</artifactId>
-    <version>0.1.2-SNAPSHOT</version>
+    <artifactId>azure-spring-boot-starter-bom</artifactId>
+    <version>0.1.2</version>
```

Comment:
Shouldn't the version which is under development be declared as SNAPSHOT? If not can you please let us what is the process Microsoft uses.